### DISCUSSION TITLE
Is it possible to analyze PCAP logs generated by SO 2.3 on SO 16.04 using Kibana?

### DISCUSSION THREAD

**Discussion by wem0019** [Security-Onion-Solutions/securityonion#3440] [Unsupported Versions] [answered]
If so, how?

**Comment by weslambert** [comment, marked as answer]
No, not unless you manually carve pcap files from the indexed pcap in 2.3.  This is because PCAP is now indexed and is only retrieved as a traditional .pcap file on-demand.

**Reply by wem0019** [reply]
That makes sense. Thank you for your help! I really appreciate it.
